tools/refract-filter.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import argparse
+import sys
+import yaml
```

Comment:
Since this might not be installed, can we make this an optional dependency and feature since the user might only be parsing JSON?